contents/blog/introducing-hogql.md

```diff
@@ -0,0 +1,112 @@
+---
+date: 2023-05-08
+title: "Introducing SQL and HogQL access for PostHog"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+category: PostHog news
+author:
+  - joe-martin
+tags:
+  - Product updates
+  - Release notes
+featuredImage: ../images/blog/hog_ql.png
+featuredImageType: full
+---
+
+Today, we're excited to announce a major new feature which we're making available to all users as a public beta on PostHog Cloud - the ability to directly access data stored in PostHog via SQL.
+
+We've added several new ways to use SQL in PostHog - you can use expressions to enhance insights, you can use SQL to filter a revamped events list, and you can even use full SQL queries to analyse data in any way you want. 
+
+We've even created [our own SQL dialect, called HogQL](/docs/product-analytics/hogql), which enables direct access to data without the insecurities that come from using ClickHouse SQL directly.
+
+PostHog's existing insights will remain valuable for the vast majority of queries, but this gives them a massive upgrade, and the ability to create your own SQL queries provides an alternative when off-the-shelf insights don't fit the bill. 
+
+Speaking of bills, these features are free for all users while in public beta - all we ask is that you [share any feedback with us](http://app.posthog.com/home#supportModal). 
+
+## What is HogQL?
+HogQL is basically our take on SQL. 
+
+Less basically, it's a transition layer over ClickHouse SQL that we've created to empower a wide range of users, from technical product managers who need multi-property breakdowns, to engineers who need to dissect data in more nuanced ways. It's intuitive for teams who are familiar with SQL, but offers a few extra advantages and features.
+
+Unique features to HogQL include simplified access to event and person properties, as well as automatically added joins when you query fields with data on a different table, such as `events.person.properties.$browser`.
+
+You can also use aggregations, such as `count`, `min`, `minIf`, `max`, `maxIf`, `sum`, `sumIf`, `avg`, `any`, and `anyIf`. 
+
+We recommend checking the docs for a more detailed [explanation of ClickHouse SQL functions which are supported in HogQL](/manual/hogql).
+
+## HogQL breakdowns
+![HogQL breakdowns](../images/blog/array/hog_breakdown.gif)
```

Comment:
@joethreepwood Can we switch these gifs out for MP4s?

Reply:
was planning to, aye

Reply:
Could we actually show breaking something down by HogQL, instead of just opening the popup? E.g. `concat(properties.$browser, ' ', properties.$browser_version)`. Same for filters. Perhaps `properties.$screen_width < properties.$screen_height` to show only "portrait mode" users?

Reply:
One other cool use of hogql is coming with this PR: https://github.com/PostHog/posthog/pull/15429

You can now aggregate funnels on any hogql expression.